secretsauce/apps/portal/models.py

```diff
@@ -0,0 +1,27 @@
+from django.db import models
+
+# Create your models here.
+
+class Project(models.Model):
+    title = models.CharField(max_length=200)
+    organization = models.CharField(max_length=200)
+
+class PredictionModel(models.Model):
+    model_type = models.CharField(max_length=200)
+    model_name = models.CharField(max_length=200)
+
+class Hyperparameters(models.Model):
+    prediction_model = models.ForeignKey(PredictionModel, on_delete=models.CASCADE)
+
+class Weights(models.Model):
+    model_type = models.ForeignKey(PredictionModel, on_delete=models.CASCADE)
+    url = models.URLField(max_length = 200) 
+
+class DataBlock(models.Model):
+    project = models.ForeignKey(Project, on_delete=models.CASCADE)
+
+class ConstraintBlock(models.Model):
+    project = models.ForeignKey(Project, on_delete=models.CASCADE)
+
+class Constraint(models.Model):
+    constraint_block = models.ForeignKey(ConstraintBlock, on_delete=models.CASCADE)
```

Comment:
Can we add a `def __str__` method here? This sets the logic for converting every model object instance into a string whenever it needs to be printed.
e.g. I created a project where 
`x=Project.create('funny project')`. 
I defined the string method as 
```
def __str__(self):
        return self.title
```
When i `print(x)`, instead of appearing as `Project Object (93249)', it will appear as `funny project`

Makes it look nicer on django admin too